### ISSUE TITLE
Add submission spot for different fish

### ISSUE THREAD

**Issue by jacobvalenzuela** [baliga-lab/aqxWeb#4]
Needs a submission of spot for different fish

**Comment by weiju** [comment]
do you mean a way to suggest new fish species ?

**Comment by jacobvalenzuela** [comment]
Yes, I think if we can choose from a large list or if we can submit a fish species.  Example, right now I have catfish in my AQX system, how can I submit that species?

**Comment by weiju** [comment]
I think we could replace the select box with an auto complete box that based on the input will suggest a list of fish to autocomplete and alternatively will allow to add a new species. We might want to add a moderation option, so only users who entered that species will see their input until it is approved for everyone by an administrator.

**Comment by jacobvalenzuela** [comment]
I think thats probably the best approach

Jacob J. Valenzuela, Ph.D.
Institute for Systems Biology
401 Terry Ave N
Seattle WA 98109
(206)-732-1496
https://www.systemsbiology.org
Jake.Valenzuela@systemsbiology.org
